<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="ScaleArea" name="scaleArea" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="SettingPage" name="settingPage" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="CoreWar">
      <property name="text">
       <string>CoreWar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="CaravanPage" name="caravanPage" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="bonus">
      <property name="text">
       <string>bonus</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScaleArea</class>
   <extends>QWidget</extends>
   <header>scalearea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingPage</class>
   <extends>QWidget</extends>
   <header>settingpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaravanPage</class>
   <extends>QWidget</extends>
   <header>caravanpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
